app/src/main/java/com/example/cinemapp/ui/main/movie_details/MovieDetailsFragment.kt

```diff
@@ -97,11 +217,22 @@ class MovieDetailsFragment : Fragment() {
             }
 
             rvMedia.adapter = mediaAdapter
-            rvMedia.layoutManager = LinearLayoutManager(context, LinearLayoutManager.HORIZONTAL, false)
+            rvMedia.layoutManager =
+                LinearLayoutManager(context, LinearLayoutManager.HORIZONTAL, false)
             PagerSnapHelper().attachToRecyclerView(rvMedia)
         }
     }
 
+    private fun setupVisibility() {
+        with(binding) {
+            clContent.visibility = View.INVISIBLE
```

Comment:
This whole content visibility could be setup through observing viewmodels state...but we will talk about that (you can skip this comment)